config/application.rb

```diff
@@ -26,6 +26,9 @@ class Application < Rails::Application
     config.autoload_paths << Rails.root.join('lib')
     config.autoload_paths << Rails.root.join('lib/integrations')
 
+    # to load from i18n-js
+    config.middleware.use I18n::JS::Middleware
```

Comment:
yes because it generates a separated JS file for the i18n/translations.js to load. But it's cached as a static file in public/assets so I won't worry too much.

The old locales.js.erb was working fine, but for some reason I was updating locale files but they were not being reflected in the generated locales.js file. Tried many times and nothing, so I gave up and used the middleware instead and it's working fine now.